tests/unit/test_cloned_project.py

```diff
@@ -0,0 +1,67 @@
+from unittest import mock
+
+from ogr.abstract import GitProject
+from packit.cloned_project import ClonedProject
+
+
+def test_cloned_project_name():
+    project = ClonedProject(full_name="namespace/repository_name")
+    assert project.repo == "repository_name"
+    assert project.namespace == "namespace"
+    assert project.full_name == "namespace/repository_name"
+
+
+def test_cloned_project_repo_namespace():
+    project = ClonedProject(repo="repository_name", namespace="namespace")
+    assert project.repo == "repository_name"
+    assert project.namespace == "namespace"
+    assert project.full_name == "namespace/repository_name"
+
+
+@mock.patch.object(GitProject, "get_git_urls", return_value={"git": "ssh url"})
```

Comment:
in all other projects we are using flexmock for mocking; would be nice if all of us agreed on a single library; Franto, would you have time to do some brief research on this? I'll add it to the related card for this sprint.

Reply:
OK, I found it interesting.

Reply:
I'd like to give the flexmock a try so I moved the tests to flexmock. Looks good!

Reply:
Finally, I've used:
- flexmock for object mocking
- mock for mocking imports